history-service/middlewares/cors.js

```diff
@@ -0,0 +1,31 @@
+import cors from "cors";
+
+const allowedOrigins = [
+  "http://localhost:5173", // frontend dev
+  "http://peerprep.s3-website-ap-southeast-1.amazonaws.com", // frontend prod
+  "http://peerprep-frontend-bucket.s3-website-ap-southeast-1.amazonaws.com", // frontend staging
+  "http://localhost:8000",
+  "http://localhost:8001",
+  "http://localhost:8002",
+  "http://localhost:8004",
+  "http://localhost:8005",
```

Comment:
We might need to update the rest of the services with the new CORS endpoints here! (just pinning it here so that we remember to do so in the future)